### PR TITLE
player playground changes

### PR DESCRIPTION
**Pull request by kishorens**
Features :

- Support for testing various player and source configurations.
- Support for dynamically generating player and source configurations.

Screenshot 2022-05-24 at 11 26 38 AM
Screenshot 2022-05-24 at 11 26 56 AM
Screenshot 2022-05-24 at 11 27 16 AM

### CODE REVIEW

File: player/player-playground/js/script.js [hunk 1]
```diff
@@ -0,0 +1,2550 @@
...
+    const limit = parts.length - 1;
+    for (let i = 0; i < limit; ++i) {
+        const key = parts[i];
+        object = object[key] ?? (object[key] = {});
+    }
+    const key = parts[limit];
+    object[key] = value;
+}
+
+function intialize() {
```
Comment (bitsmith1010) [inline player/player-playground/js/script.js:172]: replace "intialize" with "initialize"?

**Review by bitsmith1010 — CHANGES_REQUESTED** [review]
this is great! only minor comments, do you mind to separate the json objects such as `getPlayerSchema()` to separate files? also one minor type mistake (see comment)

File: player/player-playground/js/script.js (hunk 1 above)
Comment (kishorens) [reply player/player-playground/js/script.js]: Thanks, typo corrected

**Comment by kishorens** [comment]
Thanks a lot for reviewing @bitsmith1010.
**Regarding separate files**:- The demo framework that we use for demos dynamically executes the files individually, so splitting the functions into different files is not possible with the existing setup.

**Comment by bitsmith1010** [comment]
I see.. it's surprising.. yes also the js files are not loaded from index.html load the script.. this could be something to look at in the demo framework in some time..

**Review by bitsmith1010 — APPROVED** [review]
Great!
possibly in the future to try more to divide the script to several files

**Comment by kishorens** [comment]
Thanks @bitsmith1010 . Assigning the function to the window object worked, I separated the files and pushed the changes.

**Review by bitsmith1010 — APPROVED** [review]
@kishorens  now even better!

I see the CI deployed it in staging but the message is in the automated pull request https://github.com/bitmovin-engineering/demo-framework/pull/103 

tested the demo manually briefly and it looks good

**Pull request merged**